server/game/cards/05-UotE/MakeAnOpening.js

```diff
@@ -0,0 +1,34 @@
+const DrawCard = require('../../drawcard.js');
+
+class MakeAnOpening extends DrawCard {
+    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+        this.action({
+            title: 'Give -X/-X to opposing character, where X is the difference between current honor dial bid values',
+            condition: context =>
+                this.game.isDuringConflict() &&
+                this.game.currentConflict.getNumberOfParticipantsFor(context.player) >= 1 &&
+                this.game.currentConflict.getNumberOfParticipantsFor(context.player.opponent) >= 1 &&
+                context.player.showBid &&
+                context.player.opponent.showBid &&
+                context.player.showBid !== context.player.opponent.showBid,
+            target: {
+                cardType: 'character',
+                controller: 'opponent',
+                cardCondition: card =>
+                    card.isParticipating() &&
+                    (card.militarySkill > 0 || card.politicalSkill > 0),
+                gameAction: ability.actions.cardLastingEffect(context => ({
+                    effect: ability.effects.modifyBothSkills(() => -(this.getHonorDialDifference(context)))
+                }))

```

Comment:
This should be a static effect, not a dynamic one I think.  The value of X is locked in when the ability resolves - if dials change later in the conflict, the bonus shouldn't change